README.md

```diff
@@ -25,7 +25,7 @@ See [example](example.js)
 See the language independent [ProtoDef](https://github.com/ProtoDef-io/ProtoDef) specification.
 
 * [api.md](doc/api.md) documents the exposed functions and classes
-* [datatypes.md](https://github.com/ProtoDef-io/ProtoDef/blob/master/doc/datatypes.md) documents the default datatypes provided by Protodef.
```

Comment:
No this is wrong, the doc is not in node-protodef repo

Reply:
yes I'm blind ik